searx/templates/oscar/preferences.html

```diff
@@ -280,6 +280,17 @@ <h3>{{ _('Privacy') }}</h3>
                         {{ preferences_item_footer(image_proxy_info, image_proxy_label, rtl) }}
                         {% endif %}
 
+                        {% if 'query_in_title' not in locked_preferences %}
+                        {% set query_in_title_label = _('Query in the page\'s title') %}
+                        {% set query_in_title_info = _('When enabled, the result page\'s title contains your query. Your browser can record this title') %}
```

Comment:
While I was in translation I recognized that we have this sentence twice, here in oscar (missing the final dot) and in simple (with a dot at the end).

Reply:
fixed in 3f3097c7